solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -379,6 +397,23 @@ public void doFilter(ServletRequest _request, ServletResponse _response, FilterC
         }
       }
 
+      SpanContext parentSpan = GlobalTracer.extract(request);
```

Comment:
Shouldn't this be in an other filter class earlier in the filter chain? (e.g. TracingFilter)

Reply:
I don't think we should do that, since introducing another Filter require changes and several places (web.xml).